dotnet/src/IntegrationTests/Connectors/OpenAI/OpenAITextEmbeddingTests.cs

```diff
@@ -69,29 +66,4 @@ public async Task AzureOpenAITestAsync(string testInputString)
         Assert.Equal(AdaVectorLength, singleResult.Length);
         Assert.Equal(3, batchResult.Count);
     }
-
-    #region internals
-
-    private readonly RedirectOutput _testOutputHelper;
-
-    public void Dispose()
-    {
-        this.Dispose(true);
-        GC.SuppressFinalize(this);
-    }
-
-    ~OpenAITextEmbeddingTests()
```

Comment:
When you get a chance, can you also approve https://github.com/microsoft/semantic-kernel/pull/4432? It'll conflict with this.